packages/sampler/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact Sampler, newest changes on the top.
 
+## [unreleased]
+
+### Added
+
+- `Scroller` qa-sample to verify it with resizable item
```

Comment:
How about?
- `moonstone/Scroller` qa-sampler sample with resizable items

Reply:
No problem

Reply:
fwiw, we haven't been putting test changes into the CHANGELOG.  Particularly for QA, as it's not customer-facing.

Reply:
@webOS101 
Do you think that doesn't need to add CHANGELOG?

Reply:
I do not think it needs one.